dao/src/main/java/greencity/entity/order/TariffsInfo.java

```diff
@@ -60,4 +47,32 @@ public class TariffsInfo {
 
     @Column(nullable = false)
     private LocalDate createdAt;
+
+    @Column(name = "min_amount_of_big_bags")
```

Comment:
no need to specify column name as long as field name can be resolved from column name. In this case minAmountOfBigBags can be resolved from min_amount_of_big_bags